x/wasm/keeper/querier.go

```diff
@@ -428,6 +431,11 @@ func (q GrpcQuerier) BuildAddress(c context.Context, req *types.QueryBuildAddres
 	if len(salt) == 0 {
 		return nil, status.Error(codes.InvalidArgument, "empty salt")
 	}
+
+	ctx := sdk.UnwrapSDKContext(c)
+	costBuildAddress := DefaultGasCostBuildAddress * types.DefaultGasMultiplier
```

Comment:
Is this multiplied by the gas multiplier on purpose? To me the code looks like it's just using the cosmos sdk gas meter, so that expects sdk gas as input, not cosmwasm gas.

Reply:
Yes, it makes sense to remove the Multiplier because it is related to CW gas. 
I used it to make the cost in line with humanizeAddress operation. 
I will update the pr and remove it. @chipshort do you think something like 500_000 is a good value for `DefaultGasCostBuildAddress` ?